packages/gateway/src/features/dataSeeding/index.ts

```diff
@@ -128,4 +129,5 @@ export async function seedData() {
   )
   console.log(res)
   seedCertificate(token)
+  await seedApplicationConfig()
```

Comment:
I started wondering what's the benefit of storing the config to the database? What if we would fetch the application config from country config as part of the user requesting the config, in [here](https://github.com/opencrvs/opencrvs-core/blob/297ae2f63e75a6e24cc6f2350537ceedb476843c/packages/config/src/handlers/application/applicationConfigHandler.ts#L72-L83) for instance? We could also fetch whatever is in the database and do `merge(configFromCountryConfig, configFromDatabase)`. 

What do you think @jamil314 @Zangetsu101? We did a similar thing for form config in Cameroon and it massively improved the developer experience + we wouldn't have to check if the data has already been seeded or if it needs to update by some logic

Reply:
Same applies to certificates which are notorious for annoying dev experience

Reply:
What about updating the config through UI? I think it will need some amends to work with this approach, as it expects the config to be already present in the DB and updates the data on that document